yoti_python_sdk/tests/dynamic_sharing_service/extension/test_location_constraint_extension_builder.py

```diff
@@ -4,18 +4,73 @@
 from yoti_python_sdk.dynamic_sharing_service.extension.location_constraint_extension_builder import (
     LocationConstraintExtensionBuilder,
 )
+import pytest
+
+
+def test_longitude_validation():
```

Comment:
Not essential for right now, but we can use pytest to parametrize all of these new tests, which will make them more concise (like in [test_profile.py](https://github.com/getyoti/yoti-python-sdk/blob/development/yoti_python_sdk/tests/test_profile.py#L162))
Something like:
```
@pytest.mark.parametrize(
    "longitude",
    [
        270.00,
        -181,
        "27.00",
    ],
)
...assert raises error

@pytest.mark.parametrize(
    "longitude",
    [
        180.0,
        -90,
    ],
)
...assert doesn't raise error
```